redfish-core/include/registries/openbmc.json

```diff
@@ -468,6 +468,15 @@
             "Resolution": "None.",
             "Severity": "Critical"
         },
+        "GardRecord": {
```

Comment:
nit: The building of the .hpp file alphabetizes the messages. Probably best to have the messages alphabetized in the .json file also.